src/styles.css

```diff
@@ -1,10 +1,23 @@
 body{
-  background: linear-gradient(to top left, white, aquamarine);
+  background-image: url("http://www.pngmagic.com/product_images/all-free-download-blue-gradient-background.jpg");
+  background-size: 100%;
   font-style: italic;
+  color: black;
+  font-size: 30px;
 }
 h1{
   text-decoration: underline;
 }
 p{
+  font-size: medium;
```

Comment:
Better to define styles in components, if you do not need to use them it whole application

Except `body` and `html` tags styles